lib/rails-settings/settings.rb

```diff
@@ -60,6 +60,8 @@ def get_all(starting_with = nil)
         vars.each do |record|
           result[record.var] = record.value
         end
+        vars.merge! @@defaults.slice(*(@@defaults.keys - result.keys))
```

Comment:
Ha, thank you for having a look on this. :wink:
I created [another PR](https://github.com/huacnlee/rails-settings-cached/pull/80) which should (hopefuly) fix this now.